### PR TITLE
README: Update Screenshots and Badges in ZH

### LINKED ISSUE #7986: [Branch View] [README] Screenshot Update
Update README Screenshot for Branch View after https://github.com/go-gitea/gitea/issues/7454

### PR DESCRIPTION
**Pull request by 6543**
close #7986

### CODE REVIEW

**Comment by 6543** [comment]
@techknowlogick i use one old pic: `Pull Request View` so there is no empty table feald ... if you like you can make a new one / upload this to dl.gitea.io too ?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-[简体中文](https://github.com/go-gitea/gitea/blob/master/README_ZH.md)
+[简体中文](README_ZH.md)
 
-<h1> <img src="https://raw.githubusercontent.com/go-gitea/gitea/master/public/img/gitea-192.png" alt="logo" width="30" height="30"> Gitea - Git with a cup of tea</h1>
+<h1> <img src="public/img/gitea-192.png" alt="logo" width="30" height="30"> Gitea - Git with a cup of tea</h1>
```
Comment (guillep2k) [inline README.md]: This relative path looks suspicious.

File: README.md [hunk 2]
```diff
@@ -105,9 +105,20 @@ for the full license text.
...
 |:---:|:---:|:---:|
-|![Dashboard](https://image.ibb.co/dms6DG/1.png)|![Repository](https://image.ibb.co/m6MSLw/2.png)|![Commits History](https://image.ibb.co/cjrSLw/3.png)|
-|![Branches](https://image.ibb.co/e6vbDG/4.png)|![Issues](https://image.ibb.co/bJTJSb/5.png)|![Pull Request View](https://image.ibb.co/e02dSb/6.png)|
-|![Releases](https://image.ibb.co/cUzgfw/7.png)|![Activity](https://image.ibb.co/eZgGDG/8.png)|![Wiki](https://image.ibb.co/dYV9YG/9.png)|
-|![Diff](https://image.ibb.co/ewA9YG/10.png)|![Organization](https://image.ibb.co/ceOwDG/11.png)|![Profile](https://image.ibb.co/c44Q7b/12.png)|
+|![Branches](https://dl.gitea.io/screenshots/branches.png)|![Web Editor](https://dl.gitea.io/screenshots/web_editor.png)|![Activity](https://dl.gitea.io/screenshots/activity.png)|
+|![New Migration](https://dl.gitea.io/screenshots/migration.png)|![Migrating](https://dl.gitea.io/screenshots/migration.gif)|![Pull Request View](https://image.ibb.co/e02dSb/6.png)
+![Pull Request Dark](https://dl.gitea.io/screenshots/pull_requests_dark.png)|![Diff Review Dark](https://dl.gitea.io/screenshots/review_dark.png)|![Diff Dark](https://dl.gitea.io/screenshots/diff_dark.png)|
+
+<style>
```
Comment (guillep2k) [inline README.md]: Why is style here?

Comment (6543) [reply README.md]: Because If not, one row is very smal the other wide

File: README.md (hunk 1 above)
Comment (6543) [reply README.md]: What do you mean?
i make it relative to
 * point respect branches
 * and it will work in mirrors too

File: README.md (hunk 2 above)
Comment (guillep2k) [reply README.md]: I mean, is it parsed by our markdown parser? Also, I was under the impression that the `style` element was invalid inside the body of the document.

File: README.md (hunk 1 above)
Comment (guillep2k) [reply README.md]: I must be mistaken about the intention of this PR, then. I thought that it was for Giteas's README.md? Currently Gitea's README is mainly found at:

`https://github.com/go-gitea/gitea`

So this relative image will be sought at:

`https://github.com/go-gitea/gitea/public/img/gitea-192.png` (not found)

Am I missing something here?

File: README.md (hunk 2 above)
Comment (6543) [reply README.md]: You can use html inside markdown ;)

Comment (6543) [reply README.md]: @guillep2k noticed it works localy with atom's markdown preview but ont on gitea/github

File: README.md (hunk 1 above)
Comment (6543) [reply README.md]: @guillep2k the relative README.md is working fine
-> the png is set back to absolute

**Review by lunny — APPROVED** [review]

**Review by guillep2k — APPROVED** [review]

**Comment by guillep2k** [comment]
I guess it's OK to hotlink Github for this...
```
https://raw.githubusercontent.com/go-gitea/gitea/master/public/img/gitea-192.png
```

**Pull request merged**